roles/glance/templates/etc/glance/glance-swift-store.conf

```diff
@@ -0,0 +1,11 @@
+[ref1]
+{% if (glance.store_smart|bool and swift.enabled|bool) or
+      glance.store_swift|bool  %}
```

Comment:
The loading of `ref1` is already put into a if statement. If the conditional isn't true, ref1 would never be used. Seems redundant to add a conditional here too.